MAVProxy/mavproxy.py

```diff
@@ -1340,8 +1340,7 @@ def quit_handler(signum = None, frame = None):
           print("Connecting to %s" % serial_list[0])
           mpstate.module('link').link_add(serial_list[0].device)
     elif not opts.master and len(serial_list) > 1:
-          print("Error: multiple possible serial ports; use --master to select a single port")
-          sys.exit(1)
+          print("Warning: multiple possible serial ports. Use console GUI to add port")
```

Comment:
The old advice here is still good.
```suggestion
          print("Warning: multiple possible serial ports. Use console GUI or 'link add'  to add port, or restart using --master to select a single port")
```

Reply:
Fixed